packages/api/src/defaultStyleOptions.ts

```diff
@@ -95,6 +95,8 @@ const DEFAULT_OPTIONS: Required<StyleOptions> = {
   hideUploadButton: false,
   microphoneButtonColorOnDictate: '#F33',
   sendBoxBackground: 'White',
+  uploadFileTypes: '*',
+  uploadMultiple: true,
```

Comment:
Should this be `undefined` by default?

Reply:
No, since the default value is `true` and `undefined` would cause it to be falsy